projects/observability/src/shared/constants/entity-metadata.ts

```diff
@@ -5,6 +5,7 @@ export interface EntityMetadata {
   icon: string;
   detailPath(id: string, sourceRoute?: string): string[];
   listPath?: string[];
+  endpointsListPath?(id: string): string[];
```

Comment:
Do we want to name this as "endpoints"? That's just a name for APIs right now. It might change later

May be a more generic name?

Reply:
this is the list path for endpoints. Generic name would be a misfit here. If we rename endpoints to something later, we'll have to refactor these as well.

Reply:
May be `apisListPath` (_if we don't want to keep it generic_) or `childListPath` (_if it makes more sense and is generic_)

Reply:
entityMetadata map could be extended to be used with other pages as well. So, childListPath would become ambiguous, if we want to add another child list path here in the future. I'm good with apisListPath, but isn't that the same thing as apisListPath?

We should lean towards using one name. I was assuming we are calling it endpoints hereafter everywhere.

Reply:
We are actually calling APIs as "APIs" everywhere. All the component classes, EntityTypes etc. Names will keep changing IMO

Probably want to be consistent everywhere

Reply:
In that case, I prefer `apisListPath` as well. Updating.

Reply:
@skjindal93 Done.